include/message_filters/cache.h

```diff
@@ -127,11 +128,11 @@ class Cache : public SimpleFilter<M>
    */
```

Comment:
better clean it for `ROS_ERROR` like `//ROS_ERROR("Cannot set max_size to 0") ;` at line 109